package.json

```diff
@@ -54,10 +54,11 @@
     "vitest": "^1.6.0"
   },
   "dependencies": {
-    "@neondatabase/api-client": "1.9.0",
+    "@neondatabase/api-client": "^1.10.0",
     "@segment/analytics-node": "^1.0.0-beta.26",
     "axios": "^1.4.0",
     "axios-debug-log": "^1.0.0",
+    "bun": "^1.1.21",
```

Comment:
don't add bun to necessary dependencies, it's not required to run `neonctl`

Reply:
fixed